### ISSUE TITLE
Yolo-pose with coco-wholebody

### ISSUE THREAD

**Issue by xzxorb** [ultralytics/ultralytics#14899]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

HI, I'm currenctly using yolov8-pose to do wholebody pose estimation task, I tried to use coco-wholebody dataset, but the result is bad, the pose precision for hand and foor is even 0, is that possible or my training method is not fesible.

**Comment by pderrenger** [comment]
@xzxorb hi,

Thank you for reaching out. To assist you better, please ensure you are using the latest version of the Ultralytics YOLO package. If the issue persists, it might be related to the dataset format or training configuration. Make sure your COCO-WholeBody dataset is correctly formatted according to the Ultralytics YOLO pose format. Additionally, verify that your YAML configuration file accurately reflects the dataset structure and keypoints.

If you continue to face issues, please provide more details about your training setup and any specific errors or warnings you encounter. This will help in diagnosing the problem more effectively.

**Comment by Skillnoob** [comment]
@xzxorb Can you share the code used for training?

**Comment by xzxorb** [comment]
`yolo train data=coco-wholebody.yaml model=yolov8m-pose.pt epochs=500 lr0=0.01 device=0 name=train`

coco-wholebody.yaml
```
path: D:/Desktop/work/HPE/ultralytics-main/datasets/coco-wholebody # dataset root dir
train: images/train # train images (relative to 'path') 4 images
val: images/val # val images (relative to 'path') 4 images
test: # test images (optional)

# Keypoints
kpt_shape: [68,3]
# Classes
names:
  0: body
  1: face
  2: lefthand
  3: righthand
  4: foot
```

**Comment by pderrenger** [comment]
Thanks for sharing your training command and YAML configuration. It looks like your setup is generally correct. However, the poor performance might be due to several factors, such as dataset quality, annotation accuracy, or hyperparameter settings.

First, ensure your dataset annotations are accurate and follow the Ultralytics YOLO format. Verify that the keypoints are correctly labeled and visible in the images. Additionally, consider experimenting with different learning rates, batch sizes, and other hyperparameters to find the optimal settings for your specific dataset.

If the issue persists, try training with a smaller subset of your data to quickly iterate and debug. Also, ensure you are using the latest version of the Ultralytics YOLO package to benefit from any recent improvements or bug fixes.

**Comment by Skillnoob** [comment]
Try removing the `lr0` argument, as ultralytics automatically determines it for the optimizer.

**Comment by Skillnoob** [comment]
Also, if your images are too high resolution you might want to adjust the `imgsz` parameter

**Comment by xzxorb** [comment]
So what `imgze` will be better, my image's original resolution is about `640x480`, and hand and foot is quite small in the image

**Comment by Skillnoob** [comment]
then the default of 640 is fine, since you don't want to go over the maximum resolution of your images. But as I said in my first message remove the `lr0` argument and try again.

**Comment by xzxorb** [comment]
OK, thank you, I will try it later.

**Comment by pderrenger** [comment]
@xzxorb great! Let us know how it goes. If you encounter any further issues, feel free to reach out. We're here to help!
